components/create/Review.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import Chip from '@mui/material/Chip';
+import Stack from '@mui/material/Stack';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { Typography } from '@mui/material';
+import { useContext } from 'react';
+import { Divider } from '@mui/material';
+import { IPortfolio } from 'models/data/IPortfolio';
+
+const Review = () => {
+  const { title, name, headshot, about, work, skills, projects, contact } =
+    useContext(CreatePortfolioContext);
+  const portfolio = {
+    content: { title, name, headshot, about, work, skills, projects, contact },
+  } as IPortfolio;
```

Comment:
Review could accept an IPortfolio as a prop, so that Review doesn't use the context at all, and this data can always be passed as a prop. This way we can use it for both review and view page :)